### PR TITLE
test: add action sheet e2e tests

### PR DESCRIPTION
**Pull request by Seamoo13**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/4675

#### Please provide a brief summary of this pull request.
add action sheet e2e tests - core

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by nikvalor — APPROVED** [review]

File: e2e/wdio/core/pages/action-sheet.po.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    actionSheetList = 'fd-action-sheet-body ul';
+    actionSheetListItemButtons = 'fd-action-sheet-body button'
+    alertMessage = 'fd-message-toast'
+
+    open(): void {
+        super.open(this.url);
+        waitForElDisplayed(this.root);
+        waitForPresent(this.title);
+    }
+
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/pages/action-sheet.po.ts:18]: remove empty line

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    it('should check action sheet items are clickable', () => {
+        const actionSheetCount = getElementArrayLength(actionSheetMenuButton);
+
+        for (let i = 0; actionSheetCount > i; i++) {
+            click(actionSheetMenuButton, i);
+            checkElArrIsClickable(actionSheetListItemButtons);
+        }
+    });
+
+    it('should check comfy and cozy properties', () => {
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/action-sheet.e2e-spec.ts:43]: Maybe check the actual size or create a screenshot?

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        click(actionSheetMenuButton);
+        expect(getAttributeByName(actionSheetList, classAttribute)).toContain(compactValue);
+        click(actionSheetMenuButton, 1);
+        expect(getAttributeByName(actionSheetList, classAttribute)).not.toContain(compactValue);
+    });
+
+    it('should check alert appears after selection', () => {
+        click(actionSheetMenuButton);
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/action-sheet.e2e-spec.ts:51]: - why only one option? 
-  alert has option specific text that can be checked

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        click(actionSheetListItemButtons);
+        expect(waitForElDisplayed(alertMessage)).toBe(true);
+    });
+
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/action-sheet.e2e-spec.ts:55]: why only one example tested?

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    it('should check orientation', () => {
+        actionSheetPage.checkRtlSwitch();
+    });
+
+    it('should check basic visual regression', () => {
+        actionSheetPage.saveExampleBaselineScreenshot('action-sheet');
+        expect(actionSheetPage.compareWithBaseline('action-sheet')).toBeLessThan(1);
+    });
+
+    it('should check action sheet items visual regression', () => {
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/action-sheet.e2e-spec.ts:65]: options and action button state (hover/focus/active) are missing

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+        for (let i = 0; actionSheetCount > i; i++) {
+            click(actionSheetMenuButton, i);
+            checkElArrIsClickable(actionSheetListItemButtons);
+        }
+    });
+
+    it('should check comfy and cozy properties', () => {
+        click(actionSheetMenuButton);
+        expect(getAttributeByName(actionSheetList, classAttribute)).toContain(compactValue);
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/action-sheet.e2e-spec.ts:45]: maybe we can create a separate method to verify class (optional)
Something like getClassAttribute

**Review by AntonOlkhovskyi — CHANGES_REQUESTED** [review]

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts (hunk 2 above)
Comment (Seamoo13) [reply e2e/wdio/core/tests/action-sheet.e2e-spec.ts:43]: I already take the screenshot of these action sheets in "should check action sheet items visual regression"

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts (hunk 6 above)
Comment (Seamoo13) [reply e2e/wdio/core/tests/action-sheet.e2e-spec.ts:45]: I think if we do that then we should do it for all the ones we regularly use and update all the tests. Maybe better to do that in a future PR.

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts (hunk 5 above)
Comment (Seamoo13) [reply e2e/wdio/core/tests/action-sheet.e2e-spec.ts:100]: added

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts (hunk 3 above)
Comment (Seamoo13) [reply e2e/wdio/core/tests/action-sheet.e2e-spec.ts]: added

File: e2e/wdio/core/tests/action-sheet.e2e-spec.ts (hunk 4 above)
Comment (Seamoo13) [reply e2e/wdio/core/tests/action-sheet.e2e-spec.ts:90]: added

File: e2e/wdio/core/pages/action-sheet.po.ts (hunk 1 above)
Comment (Seamoo13) [reply e2e/wdio/core/pages/action-sheet.po.ts]: done

File: wdio-local.conf.js [hunk 7]
```diff
@@ -291,6 +291,13 @@ exports.config = {
             }, this);
         }, true);
 
+        browser.addCommand('addIsActiveClass', function() {
+            browser.execute(function(domElement) {
+                domElement.classList.add('is-active');
+
```
Comment (EvilAlexei) [inline wdio-local.conf.js:297]: Nit: Could you please remove this empty line?

File: wdio.conf.js [hunk 8]
```diff
@@ -333,6 +333,13 @@ exports.config = {
             }, this);
         }, true);
 
+        browser.addCommand('addIsActiveClass', function() {
+            browser.execute(function(domElement) {
+                domElement.classList.add('is-active');
+
```
Comment (EvilAlexei) [inline wdio.conf.js:339]: Nit: Could you please remove this empty line?

**Review by EvilAlexei — APPROVED** [review]

File: wdio-local.conf.js (hunk 7 above)
Comment (Seamoo13) [reply wdio-local.conf.js]: done

File: wdio.conf.js (hunk 8 above)
Comment (Seamoo13) [reply wdio.conf.js]: done

**Review by AntonOlkhovskyi — APPROVED** [review]

**Pull request merged**